src/electos/ballotmaker/cli.py

```diff
@@ -1,28 +1,52 @@
 """ The BallotMaker Command Line Interface (CLI)"""
-
+from pathlib import Path
 from typing import Optional
-
 import typer
 
-from electos.ballotmaker import __version__
+from electos.ballotmaker import make_ballots
+from electos.ballotmaker.constants import NO_ERRORS, VERSION, PROGRAM_NAME
 
-PROGRAM_NAME = "BallotMaker"
+VERSION_HELP = "Print the version number."
+EDF_HELP = "EDF file with ballot data (JSON format)"
+PDF_OUTPUT_HELP = "EDF file with ballot data (JSON format)"
+STYLE_HELP = "Stylesheet file for ballot generation"
 
-# for this to work, main needs to be a callback, not a command
+
+def version_callback(value: bool):
+    if value:
+        typer.echo(f"{PROGRAM_NAME} version: {VERSION}")
+        raise typer.Exit()
+
+
+# to display help when no arguments are provided,
+# main needs to be a callback, not a command
 app = typer.Typer(no_args_is_help=True)
```

Comment:
Does `no_args_is_help` actually _do_ anything? I don't see it in the docs, and it's not clearly doing anything in the source 🤔.

Reply:
Yes, this is what triggers the display of help text if no arguments or options are passed, like so:

```
➤ ballotmaker
Usage: ballotmaker [OPTIONS] COMMAND [ARGS]...

Options:
  --version                       Print the version number. 
...
```

I even wrote a test for this in test_cli.py: `test_usage()`